editor/gui/editor_file_dialog.cpp

```diff
@@ -842,6 +843,9 @@ void EditorFileDialog::update_file_list() {
 	dirs.sort_custom<NaturalNoCaseComparator>();
 	files.sort_custom<NaturalNoCaseComparator>();
 
+	HashMap<String, Color> folder_colors = FileSystemDock::get_singleton()->get_folder_colors();
```

Comment:
```suggestion
	const HashMap<String, Color> &folder_colors = FileSystemDock::get_singleton()->get_folder_colors();
```
For these as well in that case